core/client/src/main/java/alluxio/client/file/URIStatus.java

```diff
@@ -71,10 +71,21 @@ public long getFileId() {
   }
 
   /**
+   * DEPRECATED SINCE 1.2 AND WILL BE REMOVED IN 2.0.
+   *
    * @return the group that owns the entity referenced by this uri, mutable
+   * @deprecated @see {@link #getGroup()}
    */
+  @Deprecated
   public String getGroupName() {
-    return mInfo.getGroupName();
+    return getGroup();
+  }
+
+  /**
+   * @return the group that owns the entity referenced by this uri, mutable
```

Comment:
if think the mutable / immutable does not refer to the returned value but to the alluxio state